wwwroot/languages/en/translation.json

```diff
@@ -716,7 +716,8 @@
       "czml": "CZML",
       "gpx": "GPX",
       "json": "Terria Catalog",
-      "carto": "Carto",
+      "carto": "Carto V1",
```

Comment:
does the key "carto" need to be renamed "cartov1" throughout?

Reply:
I have kept it `"carto"` so it matches `CartoMapV1CatalogItem.type`

Reply:
At some point we will want to swap them over - but that will break existing API